test/pixel_proj_err_test.cc

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright(c) 2019 Netflix, Inc.
+ * SPDX - License - Identifier: BSD - 2 - Clause - Patent
+ */
+
+/******************************************************************************
+ * @file pixel_proj_err_test.cc
+ *
+ * @brief Unit test of pixel projection error:
+ *
+ * - av1_lowbd_pixel_proj_error_avx2
+ * - av1_highbd_pixel_proj_error_avx2
+ *
+ * @author Cidana-Edmond
+ *
+ ******************************************************************************/
+
+#include "gtest/gtest.h"
+#include "aom_dsp_rtcd.h"
+#include "EbDefinitions.h"
+#include "EbUtility.h"
+#include "EbRestoration.h"
+#include "random.h"
+#include "util.h"
+
+#define MAX_DATA_BLOCK 384
+
+namespace {
+using std::make_tuple;
+using svt_av1_test_tool::SVTRandom;
+
+static const int min_test_times = 10;
+
+typedef int64_t (*pixel_proj_fcn)(const uint8_t* src8, int32_t width,
```

Comment:
Type should be Camel_case, like PixelProjFunc

Reply:
done